EasyInventory/src/main/resources/templates/inventory.html

```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
-<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+<html xmlns:th="http://www.thymeleaf.org" lang="">
 <head>
     <meta charset="UTF-8">
 <!--    <title>Inventory</title>-->
     <link rel="stylesheet" th:href = "@{/css/inventory.css}">
+    <title></title>
```

Comment:
Title was missing

Reply:
Now, the tags are present, and the title is missing.  As long as you add the tag, might as well add a title.